config/config.go

```diff
@@ -517,3 +522,12 @@ func getAuthFromEnv(prefix string) Auth {
 	auth.CAFile = strings.TrimSpace(getDefaultString(prefix+EnvAuthSuffixCAFile, ""))
 	return auth
 }
+
+// IsIstioNamespace returns true if the namespace is the default istio namespace or an Istio component namespace
+func IsIstioNamespace(namespace string) bool {
+	if namespace == configuration.IstioNamespace {
+		return true
+	}
+	_, found := configuration.IstioNamespaces[namespace]
```

Comment:
I think this is backwards. The key is the component name, the value is the namespace. You want to see if 
"namespace" is a value anywhere in any key of IstioNamespaces (which, IMO, should be called IstioComponentNamespaces)

Reply:
yeah, it's wrong.  will fix.